istioctl/pkg/describe/describe.go

```diff
@@ -910,12 +928,58 @@ func printVirtualService(writer io.Writer, vs *clientnetworking.VirtualService,
 	if facts == 0 {
 		// We printed nothing other than the name.  Print something.
 		if len(vs.Spec.Http) > 0 {
-			fmt.Fprintf(writer, "   %d HTTP route(s)\n", len(vs.Spec.Http))
+			fmt.Fprintf(writer, "%s%d HTTP route(s)\n", printSpaces(initPrintNum+printLevel1), len(vs.Spec.Http))
 		}
 		if len(vs.Spec.Tcp) > 0 {
-			fmt.Fprintf(writer, "   %d TCP route(s)\n", len(vs.Spec.Tcp))
+			fmt.Fprintf(writer, "%s%d TCP route(s)\n", printSpaces(initPrintNum+printLevel1), len(vs.Spec.Tcp))
+		}
+	}
+}
+
+func isSubsetOf(sub map[string]string, parent map[string]string) bool {
```

Comment:
Check `pkg/maps`, may have a similar usage.

Reply:
yes, already have a similar methods of maps.Contains